packages/scripts/scripts/check-licenses.js

```diff
@@ -0,0 +1,119 @@
+// Do this as the first thing so that any code reading it knows the right env.
+process.env.BABEL_ENV = 'development';
```

Comment:
Those 10 lines at the beginning are Jest specific - they can be removed probably.

Reply:
Ah, cool. I assumed they were necessary, but I'll get rid of them.